lib/persisted-model.js

```diff
@@ -483,6 +495,19 @@ module.exports = function(registry) {
   };
 
   /**
+   * Replace set of attributes.  Performs validation before replacing.
+   *
+   * @param {Object} data Data to replace.
+   * @callback {Function} callback Callback function called with `(err, instance)` arguments.  Required.
+   * @param {Error} err Error object; see [Error object](http://docs.strongloop.com/display/LB/Error+object).
+   * @param {Object} instance Repalced instance.
+   */
+
+  PersistedModel.replaceById = function replaceById(data, cb) {
```

Comment:
I think this methods has a different signature:

```js
function replaceById(id, data, cb);
```

Doesn't it? Please update jsdoc comment too.